packages/driver/src/cypress/error_messages.coffee

```diff
@@ -812,7 +812,14 @@ module.exports = {
     unavailable: "The XHR server is unavailable or missing. This should never happen and likely is a bug. Open an issue if you see this message."
 
   setCookie:
+    backend_error: """
+    #{cmd('setCookie')} had an unexpected error setting the requested cookie in {{browserDisplayName}}.
+
+    {{errStack}}
+    """
     invalid_arguments: "#{cmd('setCookie')} must be passed two string arguments for name and value."
+    invalid_name: "#{cmd('setCookie')} must be passed an RFC-6265-compliant cookie name. You passed:\n\n`{{name}}`"
```

Comment:
I wonder if this warning should apply to all the other cookie commands that take a 'name' option as well.

Reply:
1aabc5925